13/Dockerfile.rhel8

```diff
@@ -11,6 +11,7 @@ FROM ubi8/s2i-core
 #                           PostgreSQL administrative account
 
 ENV POSTGRESQL_VERSION=13 \
+    POSTGRESQL_PREV_VERSION=12 \
```

Comment:
The valid code for fixing this is here: https://github.com/sclorg/postgresql-container/blob/master/src/Dockerfile#L14

Please do the following steps.

1. fix file src/Dockerfile
2. commit the change.
3. The sources are generated automatically.

Reply:
Updated this file and created the new PR #456.  Thanks for pointing out.